lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1347,6 +1347,18 @@ OpFoldResult AtenIntScalarOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenIntBoolOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenIntBoolOp::fold(ArrayRef<Attribute> operands) {
```

Comment:
Can you add e2e tests for this op too?

Reply:
Done! Added three e2e tests for this op.